composer.json

```diff
@@ -10,20 +10,22 @@
     ],
     "require": {
         "jms/metadata": "~1.5",
-        "symfony/property-access": "~2.4",
+        "symfony/property-access": "~2.6||~3.0",
```

Comment:
No, you've got a good point here. Other people have to assume that stuff like this changes... but apart from that 2.6 is also dead since 01/16 😉 So I would even go to `~2.7`.